src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+	private static final String ERROR_MESSAGE = "[ERROR] 구분자 혹은 입력 값을 다시 한번 확인해주세요.";
+	private static final String BLANK = "";
+	private static final String DELIMITER = ",|:";
+	private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+
+	public static int splitAndSum(String string) {
+		if (validateNullOrBlank(string)) {
+			return 0;
+		}
+		return sum(stringArrayToIntArray(split(string)));
+	}
+
+	private static boolean validateNullOrBlank(String string) {
+		if (string == null || string.equals(BLANK)) {
+			return true;
+		}
+		return false;
+	}
+
+	private static String[] split(String string) {
+		Matcher matcher = Pattern.compile(CUSTOM_DELIMITER).matcher(string);
+		if (useCustomDelimiter(matcher)) {
+			return splitByCustomDelimiter(matcher);
+		}
+		return splitByDelimiter(string);
+	}
+
+	private static boolean useCustomDelimiter(Matcher matcher) {
+		return matcher.find();
+	}
+
+	private static String[] splitByCustomDelimiter(Matcher matcher) {
+		String customDelimiter = matcher.group(1);
+		return matcher.group(2).split(customDelimiter);
+	}
+
+	private static String[] splitByDelimiter(String string) {
+		return string.split(DELIMITER);
+	}
+
+	private static int[] stringArrayToIntArray(String[] stringArray) {
+		try {
+			int[] intArray = Arrays.stream(stringArray).mapToInt(Integer::parseInt).toArray();
```

Comment:
```suggestion
			int[] intArray = Arrays.stream(stringArray)
			.mapToInt(Integer::parseInt)
			.toArray();
```

하나의 라인에 하나의 .을 사용하면 가독성이 좋아집니다 :)